test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests.xml

```diff
@@ -24,12 +24,18 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 
 <!DOCTYPE suite SYSTEM "cmdlinetester.dtd">
 
-<suite id="J9 Command-Line Option Tests" timeout="180">
+<suite id="J9 Command-Line Option Tests" timeout="600">
     <variable name="READIPINFOFORRAS" value="-XX:+ReadIPInfoForRAS"/>
     <variable name="NOREADIPINFOFORRAS" value="-XX:-ReadIPInfoForRAS"/>
     <variable name="READIPINFOFORRAS_MESSAGE" value="enabled network query to determine host name and IP address for RAS."/>
     <variable name="NOREADIPINFOFORRAS_MESSAGE" value="disabled network query to determine host name and IP address for RAS."/>
 
+    <variable name="JVM_HEAP_LIMIT" value="-Xmx8G"/>
```

Comment:
Please use a smaller limit. Some machines may not have this amount of memory. -Xmx512k should be fine.

Reply:
Typo in my previous comment, which I edited. -Xmx512m